intezer_analyze_cli/commands.py

```diff
@@ -150,6 +151,46 @@ def analyze_by_txt_file_command(path: str):
         click.Abort()
 
 
+def index_by_txt_file_command(path: str, index_as: str, family_name: str):
+    try:
+        with open(path, 'r') as file:
+            hashes = [line.rstrip('\n') for line in file.readlines()]
+        index_exceptions = []
+        with click.progressbar(length=len(hashes),
+                               label='Uploading files',
+                               show_pos=True,
+                               width=0) as upload_progress:
+            for sha256 in hashes:
+                try:
+                    index_hash_command(sha256, index_as, family_name, index_exceptions)
+                except sdk_errors.HashDoesNotExistError:
+                    index_exceptions.append('Hash: {} does not exist in the system'.format(sha256))
+                except sdk_errors.IntezerError as e:
+                    index_exceptions.append('Error occurred with hash: {}'.format(sha256, e))
+                upload_progress.update(1)
+
+        for exception in index_exceptions:
+            click.echo(exception)
+
+        if default_config.is_cloud:
+            click.echo('index updated. In order to check their results, go to: {}'
+                       .format(default_config.index_results_url))
+        else:
+            click.echo(
+                'index updated. In order to check the results go to Private Indexed Files under Analysis Reports')
+    except IOError:
+        click.echo('No read permissions for {}'.format(path))
+        click.Abort()
+
+
+def index_hash_command(sha256: str, index_as: str, family_name: Optional[str], index_exceptions):
+    try:
+        index = Index(index_as=sdk_consts.IndexType.from_str(index_as), sha256=sha256, family_name=family_name)
+        index.send(wait=True)
```

Comment:
We still need to fix this - `send(wait=False)`
After that we need to call `wait_for_completion` for each one